### PR TITLE
feat(perf-summary): adding user misery bar

### PR DESCRIPTION
**Pull request by doralchan**
**Before:**
Screen Shot 2020-06-12 at 10 03 31 AM
Screen Shot 2020-06-12 at 10 03 21 AM

**After:**
Screen Shot 2020-06-12 at 10 10 12 AM
Screen Shot 2020-06-12 at 10 02 14 AM

### CODE REVIEW

File: src/sentry/static/sentry/app/components/userMisery.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    '[affectedUsers] out of [totalUsers] ([miseryPercentage]%) unique users waited more than [duration]ms',
+    {
+      affectedUsers: miserableUsers,
+      totalUsers,
+      miseryPercentage,
+      duration: 4 * miseryLimit,
+    }
+  );
+  return (
+    <Tooltip title={title} disabled={false} containerDisplayMode="block">
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/userMisery.tsx:34]: You shouldn't need `disabled=false` as that is the default value.

File: src/sentry/static/sentry/app/views/performance/transactionSummary/userStats.tsx [hunk 2]
```diff
@@ -40,17 +48,39 @@ class UserStats extends React.Component<Props> {
...
       <Container>
         <div>
-          <StatTitle>{t('Apdex Score')}</StatTitle>
+          <SectionHeading>{t('Apdex Score')}</SectionHeading>
           <StatNumber>{!stats ? '\u2014' : stats['apdex()']}</StatNumber>
         </div>
         <div>
-          <StatTitle>{t('User Misery')}</StatTitle>
-          <StatNumber>{!stats ? '\u2014' : stats['user_misery(300)']}</StatNumber>
+          <SectionHeading>{t('Baseline Duration')}</SectionHeading>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/performance/transactionSummary/userStats.tsx:77]: I wouldn't include Baseline Duration until we have data to put here (which could be a while as we haven't really defined what baseline is going to be yet).

Comment (doralchan) [reply src/sentry/static/sentry/app/views/performance/transactionSummary/userStats.tsx:79]: Hid it!

**Review by dashed — APPROVED** [review]
ok

**Pull request merged**